code/sarix_model/sarix_model.py

```diff
@@ -46,7 +47,7 @@ def get_sarix_preds(model_config, run_config):
     df['xmas_spike'] = np.maximum(3 - np.abs(df['delta_xmas']), 0)
     
     xy_colnames = ["inc_trans_cs"] + model_config.x
-    batched_xy = df[xy_colnames].values.reshape(len(df['location'].unique()), -1, 2)
+    batched_xy = df[xy_colnames].values.reshape(len(df['location'].unique()), -1, len(xy_colnames))
```

Comment:
fixing a bug that surfaced when `model_config.x = []`; last dimension should just be 1 in that case.